### PR TITLE
Case 21157: Fix Desktop.qml crash on local mac build

### PR DESCRIPTION
**Pull request by danteruiz**
ticket: https://highfidelity.manuscript.com/f/cases/21157/Fix-Desktop-qml-crash-on-local-mac-builds

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17234/HighFidelity-Beta-PR14907-5e9b63b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17234/HighFidelity-Beta-PR14907-5e9b63b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17234/HighFidelity-Beta-Interface-PR14907-5e9b63b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17234/HighFidelity-Beta-PR14907-5e9b63b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17234/HighFidelity-Beta-Interface-PR14907-5e9b63b.exe

**Comment by danteruiz** [comment]
Test plan 
- mac sure the ui appear and works correct on mac and windows
- also run this test plan https://github.com/highfidelity/hifi/pull/14877

**Comment by danteruiz** [comment]
I download both the mac and windows build and everything look fine.
I have also built this PR on mac and windows and they both look fine

File: interface/resources/qml/+webengine/Browser.qml [hunk 1]
```diff
@@ -4,7 +4,7 @@ import QtWebEngine 1.5
 
 import controlsUit 1.0
 import stylesUit 1.0
-import "qrc:////qml//windows"
+import "windows"
```
Comment (wayne-chen) [inline interface/resources/qml/+webengine/Browser.qml]: shouldn't this match `QmlWebWindow.qml`'s `import "../windows"`?

File: interface/resources/qml/+webengine/InfoView.qml [hunk 2]
```diff
@@ -12,7 +12,7 @@ import QtQuick 2.5
 import Hifi 1.0 as Hifi
 
 import controlsUit 1.0
-import "qrc:////qml//windows" as Windows
+import "windows" as Windows
```
Comment (wayne-chen) [inline interface/resources/qml/+webengine/InfoView.qml]: same as above

File: interface/resources/qml/+webengine/Browser.qml (hunk 1 above)
Comment (wayne-chen) [inline interface/resources/qml/+webengine/Browser.qml]: shouldn't this match `QmlWebWindow.qml`'s `import "../windows"`?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17246/HighFidelity-Beta-PR14907-a111aa1.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17246/HighFidelity-Beta-PR14907-a111aa1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17246/HighFidelity-Beta-Interface-PR14907-a111aa1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17246/HighFidelity-Beta-PR14907-a111aa1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17246/HighFidelity-Beta-Interface-PR14907-a111aa1.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17256/HighFidelity-Beta-PR14907-562e159.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17256/HighFidelity-Beta-PR14907-562e159.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17256/HighFidelity-Beta-Interface-PR14907-562e159.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17256/HighFidelity-Beta-PR14907-562e159.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17256/HighFidelity-Beta-Interface-PR14907-562e159.exe

**Comment by danteruiz** [comment]
I retested this any everything works as expected on mac and windows

File: interface/resources/qml/hifi/tablet/+webengine/BlocksWebView.qml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import QtQuick 2.0
+import QtWebEngine 1.5
+import "../../controls" as Controls
+
+Controls.TabletWebView {
+	profile: WebEngineProfile { httpUserAgent: "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.90 Safari/537.36"}
```
Comment (AndrewMeadows) [inline interface/resources/qml/hifi/tablet/+webengine/BlocksWebView.qml]: I know you only moved this file but... that looks like a real **tab** indentation.  Do we allow those in **qml** files?

**Pull request merged**